### DISCUSSION TITLE
Wallet Security

### DISCUSSION THREAD

**Discussion by Kens8** [OneKeyHQ/app-monorepo#6118] [General]
It seems to me that if a new wallet is created on the OneKey Desktop app then the private key is stored on the PC so security is potentially compromised. Therefore it may be better just to use separate accounts attached to the main wallet which uses the private key stored on the hardware wallet. Is this correct?

**Comment by revan-zhang** [comment]
There are two main types of wallets in onekey software, HD wallets and hardware wallets. 

The HD wallet is similar to metamask in that it encrypts the private key data and stores it locally, the hardware wallet needs to be purchased and used by connecting to the onekey software, and the private key is completely stored inside the hardware.
